src/sentry/mail/adapter.py

```diff
@@ -63,8 +61,7 @@ def rule_notify(
         project = event.group.project
         extra["project_id"] = project.id
 
-        # XXX(workflow): remove the extra condition after digests.enabled(project) when wf2.0 experiment is over
-        if digests.enabled(project) and target_type != ActionTargetType.RELEASE_MEMBERS:
+        if digests.enabled(project):
```

Comment:
Do we have a digests version of the release email?

Reply:
In the case of active release notifications, I thought it might make sense to forgo batching them into a digest - at least for now. Since the motivation for ARM is to have immediate notifications come in asap. But maybe we should do it for performance reasons?

Reply:
I think that we'll need to leave this in place in that case. Otherwise these will end up going through the digest path if more than one issue occurs in the release in a short period of time

Reply:
The rollout for this feature would first require us to delete the issue alert rules I created as part of the WF2.0 experiment. So, ideally, no Issue Alert rules should be using `target_type == ActionTargetType.RELEASE_MEMBERS` at all.

Reply:
Oh, I see. I missed that we're not calling `rule_notify` here at all for these, since the actions specifically calls elsewhere. That makes sense